repos/system_upgrade/el7toel8/libraries/dnfplugin.py

```diff
@@ -106,7 +106,8 @@ def _transaction(context, stage, target_repoids, tasks, test=False):
                     stage,
                     DNF_PLUGIN_DATA_PATH
                 ],
-                callback_raw=utils.logging_handler
+                callback_raw=utils.logging_handler,
+                env=cmd_env
```

Comment:
The problem why this doesn't work is that this is a container execution and environment variables aren't forwarded. The solution to this is that we add additional support for it to the mounting.py in the libraries